src/Meilisearch/MeilisearchClient.cs

```diff
@@ -1,17 +1,16 @@
+using System;
```

Comment:
Moving these directives to the outside of the namespace could cause a breaking change for someone?

Reply:
No, this is fine for this library.

If you really want to understand why this option exists : https://stackoverflow.com/a/151560/1026105

The default nowadays is to have them outside.